docs/can-guides/commitment/recipes/credit-card-simple/credit-card-simple.md

```diff
@@ -147,15 +113,10 @@ viewModel.amount = 1000;
 
 ### The solution
 
-Update the __HTML__ tab to:
-
-@sourceref ./1-setup.html
-@highlight 10,19,22
```

Comment:
Changing the HTML is still required to get started.

I’d run through the guide again to make sure all the steps can be followed to achieve the right results.

Reply:
Done!